pkg/data/task_result.go

```diff
@@ -37,7 +38,8 @@ type ImportTaskOutput struct {
 	VideoFilePath    string `json:"videoFilePath"`
 	MetadataFilePath string `json:"metadataFilePath"`
 	// This is livepeerAPI.AssetSpec but we don't want to depend on the whole pkg
-	AssetSpec interface{} `json:"assetSpec"`
+	AssetSpec                  interface{} `json:"assetSpec"`
+	PlaybackRecordingSessionId string      `json:"playbackRecordingSessionId"`
```

Comment:
This field should be in the `AssetSpec` instead, since it is part of the asset schema. It is also supposed to be copied as is to the asset object in the API, so it should be part of the spec.

So you should be adding this field here instead: https://github.com/livepeer/go-api-client/blob/main/api.go#L224

Reply:
Got it, thanks for the pointer